common/djangoapps/util/log_sensitive.py

```diff
@@ -141,7 +138,28 @@ def cli_gen_keys():
     )
 
 
-@click.command('decrypt', help="Decrypt a logged message")
+@click.command('decrypt', help="""Decrypt a logged message.
+
+If possible, use bash process indirection to keep the private key from
+touching disk or shell history unencrypted. The safest way is to keep
+the private key in an encrypted file:
+
+    --private-key-file <(gpg2 --decrypt auth-logging-key.enc)
+
+Alternatively, you could copy it from a password manager to your
+clipboard and use a CLI clipboard tool to retrieve it:
+
+    --private-key-file <(xsel -bo) # Linux
```

Comment:
You need the `\b` if you want this formatting to stick in the click output.
```suggestion
    \b
    --private-key-file <(xsel -bo) # Linux
```

Reply:
Ah! Thanks. (And looks like it needs to be at the beginning of the line, otherwise the indentation gets added to the next line.)